pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -420,28 +420,41 @@ func poolSecurityContext(pool *miniov2.Pool, status *miniov2.PoolStatus) *v1.Pod
 
 // Builds the security context for containers in a Pool
 func poolContainerSecurityContext(pool *miniov2.Pool) *v1.SecurityContext {
+	// Default values:
+	// By default, values should be totally empty if not provided
+	// This is specially needed in OpenShift where Security Context Constraints restrict them
+	// if let empty then OCP can pick the values from the constraints defined.
+	containerSecurityContext := corev1.SecurityContext{}
 	runAsNonRoot := true
 	var runAsUser int64 = 1000
 	var runAsGroup int64 = 1000
```

Comment:
```suggestion
	var runAsNonRoot bool
	var runAsUser int64
	var runAsGroup int64
```

Reply:
I think i's better to have the defaults for true,1000 and 1000 than false,0 and 0